install.sh

```
@@ -9,7 +9,11 @@ if [ "$EUID" -ne 0 ]; then
     exit 1
 fi
 
-DEB_FILE="bastion-firewall_2.0.0_all.deb"
+if [ -f "VERSION" ]; then
+    DEB_FILE="bastion-firewall_$(tr -d '[:space:]' < VERSION)_all.deb"
+else
+    DEB_FILE=$(ls bastion-firewall_*_all.deb 2>/dev/null | head -n1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)